src/components/Ranker/Ranker.module.scss

```diff
@@ -27,6 +27,32 @@
   & + & {
     margin-left: var(--ranker-gap);
   }
+
+  .columnTitle {
+    display: flex;
+
+    button {
+      margin-left: auto;
+      background: none;
+      border: none;
+      font-size: 0.75em;
+      width: 2em;
+      cursor: pointer;
+    }
+
+    button span::after {
+      content: '△';
```

Comment:
this functions just fine but it may look better if we use icons instead - can Design provide icons? or do you want to pull in React icons much like we have in DM2 and LSP